radicle/src/cob/patch.rs

```diff
@@ -464,8 +464,8 @@ impl fmt::Display for Verdict {
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct CodeLocation {
-    /// File being commented on.
-    pub blob: git::Oid,
+    /// Path of file being commented on.
+    pub path: String,
```

Comment:
```suggestion
    pub path: PathBuf,
```

Reply:
I can't think of a reason why we'd need the blob-id if we have commit+path, right @FintanH ?

Reply:
It should be equivalent, since `commit -> tree -> path -> blob`

Reply:
Indeed! It might be worth keeping in the case of improving `surf` down the line though. Much quicker to find the blob by its id over looking up via a commit's tree.

Reply:
Right. Let's keep both then I suggest.